test/shared_setup.py

```diff
@@ -170,7 +176,12 @@ def test_prepare_rpm(self):
         setup = vsc_setup()
         setup.REPO_LIB_DIR = libdir
         setup.prepare_rpm(package)
-        self.assertEqual(setup.SHARED_TARGET['packages'], ['vsc', 'vsc.test'])
+
+        # Generate list of packages, sorted
+        package_list = list(setup.SHARED_TARGET['packages'])
+        package_list.sort()
+
+        self.assertEqual(package_list, ['vsc', 'vsc.test'])
```

Comment:
Same here, more straightforward (imho) change:

```python
self.assertEqual(sorted(setup.SHARED_TARGET['packages']), ['vsc', 'vsc.test'])
```

Reply:
I was mainly doing for readability, glad to change!